lib/utils.js

```diff
@@ -39,26 +37,22 @@ function configureBinaryLog (opts) {
  * @return {?string} The full path of cmd. `null` if the cmd is not found.
  */
 async function resolveExecutablePath (cmd) {
-  const executable = system.isWindows() ? 'where' : 'which';
-  let stdout;
+  let executablePath;
   try {
-    ({stdout} = await exec(executable, [cmd]));
-    if (stdout) {
-      const executablePath = system.isWindows() ? stdout.split(EOL)[0] : stdout.trim();
+    executablePath = await fs.which(cmd);
+    if (executablePath) {
       if (await fs.exists(executablePath)) {
         return executablePath;
       }
     }
   } catch (err) {
-    if (err.code === 1) {
-      // Expected no command error
-      // TODO: make sure what error code is raised in Windows
+    if (err.stack && err.stack.match(/not found/gi)) {
```

Comment:
does `/not found/gi.test(err.message)` work?

Reply:
Ah, yes, it has `message`... Could not find it in code...